### ISSUE TITLE
Error: force is not defined

### ISSUE THREAD

**Issue by Lythenas** [paperwm/PaperWM#505]
**Describe the bug**
I recently got the following error(s):
```
Apr 04 19:42:35 anarchy gnome-shell[1419]: JS ERROR: ReferenceError: force is not defined
                                           ensureViewport@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:3010:55
                                           layout@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:564:27
                                           setMonitor@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:1479:14
                                           _animateToSpaceOrdered/<@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:2064:19
                                           _animateToSpaceOrdered@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:2062:23
                                           _initWorkspaceSequence@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:2038:14
                                           selectSequenceSpace@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:2106:18
                                           selectDownSpace@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:3787:12
                                           _doAction@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:206:20
                                           show@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:116:14
                                           preview_navigate@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:446:14
                                           asKeyHandler/<@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/keybindings.js:272:20
Apr 04 19:42:36 anarchy gnome-shell[1419]: JS ERROR: ReferenceError: force is not defined
                                           ensureViewport@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:3010:55
                                           layout@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:564:27
                                           setMonitor@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:1479:14
                                           destroy@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:348:24
                                           _finish@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:226:31
                                           _keyReleaseEvent@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:183:22
Apr 04 19:42:38 anarchy gnome-shell[1419]: Attempting to freeze the notification queue for object ClutterText[0x5631e84c4f30]; Property notification does not work during instance finalization.
Apr 04 19:42:38 anarchy gnome-shell[1419]: Attempting to thaw the notification queue for object ClutterText[0x5631e84c4f30]; Property notification does not work during instance finalization.
Apr 04 19:42:38 anarchy gnome-shell[1419]: Attempting to freeze the notification queue for object ClutterText[0x5631e84d6970]; Property notification does not work during instance finalization.
Apr 04 19:42:38 anarchy gnome-shell[1419]: Attempting to thaw the notification queue for object ClutterText[0x5631e84d6970]; Property notification does not work during instance finalization.
Apr 04 19:42:39 anarchy gnome-shell[1419]: JS ERROR: ReferenceError: force is not defined
                                           ensureViewport@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:3010:55
                                           layout@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:564:27
                                           setMonitor@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:1479:14
                                           switchWorkspace@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/tiling.js:1962:32
                                           destroy@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:364:38
                                           _finish@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:226:31
                                           _keyReleaseEvent@/home/ms/.local/share/gnome-shell/extensions/paperwm@hedning:matrix.org/navigator.js:183:22

```

This also leads to workspaces on the secondary monitor stop working. As if PaperWM was not enabled.

Primary monitor works as expected.

**To Reproduce**
1. Login
2. Overview shows, Thunderbird was autostarted
3. Press a random key (unfortunately I don't know which key I pressed)
4. Thunderbird window moves to different workspace
5. Workspaces on secondary monitor stop working. Workspaces on primary monitor still work.

**Expected behavior**
No error and workspaces keep working.

**Screenshots**

**System information:**
Please execute `./gather-system-info.sh` in you PaperWM clone and paste the output below.

```
Distribution: Arch Linux
GNOME Shell 43.3
PaperWM branch/tag: _trying_out
PaperWM commit: 3606ba087b32fb081e5c03c977767d17b945afdb
Enabled extensions:
- paperwm@hedning:matrix.org
- switcher@landau.fi
- appindicatorsupport@rgcjonas.gmail.com
```

**Additional context**
I'm mainly leaving this here to lock at (and probably fix) later.

**Comment by jtaala** [comment]
Hey @Lythenas, looks like you merged in  an earlier version of #502.

This was fixed by this commit https://github.com/paperwm/PaperWM/pull/502/commits/dda0f8979e84460869ca32755ce87c5985fa4a40 and then cleaned up (with a better approach) in this commit https://github.com/paperwm/PaperWM/pull/502/commits/672bd70836d7ea793ca8811edc51d2caa78c90ed (note these were fixed in #502 on April 1st,  and hence before this PR got merged in `develop`).

Unfortunately, the commit you're on doesn't look like it's been pushed in your fork, so I can't confirm.  But, merge in the latest `develop` (which now includes #502) and that should fix it for you.

Let me know if you run into any troubles!

**Comment by Lythenas** [comment]
Ah yes that is very possible. I will close this as I also cannot reproduce it.
